packages/docs/src/components/DocsApi/ApiDocs.vue

```diff
@@ -31,7 +31,7 @@
                 <MarkdownView :value="apiPropOption.types" />
```

Comment:
Maybe bar here instead of pointer?

![image](https://user-images.githubusercontent.com/5394573/134339411-be28d9aa-faf8-42b1-81e5-f74d1121e00c.png)